lessons/Early-Math/associative-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,112 @@
+= The Associative Property
+
+@description{This lesson introduces students to the Associative Property by way of Circles of Evaluation.}
+
+@lesson-prereqs{commutative-property}
+
+@keywords{Associative Property}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Recognize that, no matter how we group numbers being either added or multiplied, the end result will always be the same.
+* Demonstrate their understanding of the Associative Property via Circles, numeric expressions, and words.
+
+| Student-facing Lesson Goals
+|
+
+* Let's explore the Associative Property via Circles of Evaluation.
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Key Points For The Facilitator
+|
+* Just as when we introduced the Commutative Property, we asked students exclusively to manipulate the order of the values... when we introduce the Associative Property, we will ask students exclusively to manipulative the groupings of the values.
+* Students will encounter opportunities to apply both properties at the end of this lesson, and in future lessons.
+|===
+
+== The Associative Property @duration{30 minutes}
+
+=== Overview
+
+Students analyze pairs of Circles to see if they represent equivalent expressions. The Circles make visible the structural changes that occur when we apply the Associative Property.
+
+=== Launch
+
+The Associative Property defines an equivalence relation over structures, such that "adding 4 to the sum of 3 and 2" is equivalent to "adding the sum of 4 and 3 to 2." In the short term, familiarity with the Associative Property enables students to make mental computations more efficiently. In the long term, comfort with the laws of arithmetic will provide a strong foundation for more complex algebraic reasoning.
```

Comment:
I think most people will have to read this more than once in order to make sense of what you're trying to say. Any good reason not to use the words "grouping" or "regrouping" before the concrete example to give teachers a heads up of what they're looking for in the examples?

Reply:
Ack! I could've sworn I changed this! It is definitely not written in a student- or teacher-friendly way.